### PR TITLE
Chore(Docs): Added Azure Stress Docs

### PR DESCRIPTION
**Pull request by avaakash**
Signed-off-by: Akash Shrivastava <akash.shrivastava@harness.io>

# Harness Developer Pull Request
This PR will 
1. Add the docs for the Azure stress chaos experiment
2. Cleanup and fix docs for azure instance stop
3. Cleanup and fix docs for kubelet-density
4. Fix the EC2 CPU Hog diagram typo

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/chaos-engineering/chaos-faults/azure/azure-instance-cpu-hog.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+        </tr>
+        <tr>
+          <td> AZURE_AUTH_LOCATION </td>
+          <td> Provide the name of the azure secret credentials files</td>
+          <td> Defaults to <code>azure.auth</code> </td>
+        </tr>
+        <tr>
+            <td> SCALE_SET </td>
+            <td> Whether the Instance are part of ScaleSet or not. It can be either disable or enable</td>
+            <td> Defaults to `disable` </td>
```
Comment (ispeakc0de) [inline docs/chaos-engineering/chaos-faults/azure/azure-instance-cpu-hog.md]: put disable inside html code block - ` won't work inside html tables

File: docs/chaos-engineering/chaos-faults/azure/azure-instance-memory-hog.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+        <td> Whether the Instance are part of ScaleSet or not. It can be either disable or enable</td>
+        <td> Defaults to `disable` </td>
+    </tr>
+    <tr>
+        <td> MEMORY_CONSUMPTION </td>
+        <td> The amount of memory to be hogged in the azure instance in terms of mega bytes </td>
+        <td> Defaults to 500MB </td>
+    </tr>
+    <tr>
+        <td> MEMORY_CONSUMPTION </td>
```
Comment (ispeakc0de) [inline docs/chaos-engineering/chaos-faults/azure/azure-instance-memory-hog.md:124]: This ENV row is duplicate

File: docs/chaos-engineering/chaos-faults/azure/azure-instance-stop.md [hunk 3]
```diff
@@ -46,15 +51,21 @@ stringData:
       "managementEndpointUrl": "XXXXXXXXX"
     }
 ```
+
 - If you change the secret key name (from `azure.auth`) please also update the `AZURE_AUTH_LOCATION` ENV value on `experiment.yaml`with the same name.
+
 :::
 
-## Default Validations
+## Default Validationsw
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/azure/azure-instance-stop.md]:
```suggestion
## Default Validations
```

File: docs/chaos-engineering/chaos-faults/azure/azure-instance-stop.md [hunk 4]
```diff
@@ -46,15 +51,21 @@ stringData:
       "managementEndpointUrl": "XXXXXXXXX"
     }
 ```
+
 - If you change the secret key name (from `azure.auth`) please also update the `AZURE_AUTH_LOCATION` ENV value on `experiment.yaml`with the same name.
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/azure/azure-instance-stop.md]:
```suggestion
- If you change the secret key name (from `azure.auth`) please also update the `AZURE_AUTH_LOCATION` ENV value on `experiment.yaml` with the same name.
```

**Review by neelanjan00 — CHANGES_REQUESTED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**